site: add knaito to member page (#1264)
apache/cordova-docs: www/_members/knaito.md

---
name: Kenichi Naito (Ken)
subtitle:  "<strong>PMC Member</strong> since July 2018"
imageurl: https://avatars.githubusercontent.com/u/5112183
twitter: https://twitter.com/knight_9999
homepage: https://github.com/knight9999
---

> First learned about Cordova in 2012 and since then have been involved in hybrid app development, development support, and plugin development.

Reviewer: ```suggestion
Has first learned about Cordova in 2012 and has since been involved in hybrid app and plugin development and support.
```

Author: Thanks !

> Joined PMC Member in 2018 and assist with development and testing.

Reviewer 2: ```suggestion
Joined PMC in 2018 and assists with development and testing.
```

Author: Thanks!
